api/routers/project.ts

```diff
@@ -0,0 +1,14 @@
+import { getAllProjects, getProject, createProject,updateProject, deleteProject } from '../controllers/project';
+import { Router } from 'express';
+
+const ProjectRouter = Router();
+ProjectRouter.route("/").get(getAllProjects);
+ProjectRouter.route("/:project_id").get(getProject);
```

Comment:
id is enough as value instead of project_id

Reply:
/:project_id/tasklist/:tasklist_id 
i'll have something like this, might be good for consistency, but i can be wrong

Reply:
Please read this: https://enos.itcollege.ee/~jpoial/oop/naited/Clean%20Code.pdf#page=79

It looks like your repeat yourself alot

Reply:
If you truly want simplicity you should **not repeat your self** check this in the clean code book link